static/_quarto.yml

```diff
@@ -26,9 +26,8 @@ website:
         href: https://github.com/cdcepi/FluSight-forecast-hub
   page-footer:
     right:
-      - icon: gear-fill
-        text: Built with the hubDashboard App
-        href: https://github.com/apps/hubDashboard
+      - text: "Built with the hubverse dashboard"
```

Comment:
Silly question: do we consider Hubverse a proper noun? I see `hubverse` and `Hubverse` used interchangeably.

Reply:
Not a silly question at all! I think it probably should be a proper noun.

Reply:
```suggestion
      - text: "Built with the Hubverse dashboard"
```